index.js

```diff
@@ -102,10 +102,31 @@ module.exports = {
         "jsx-a11y/no-static-element-interactions": "off",
         "jsx-a11y/click-events-have-key-events": "off",
         "jsx-a11y/mouse-events-have-key-events": "off",
+        "jsx-a11y/anchor-is-valid": "off",
 
         ////////// Imports //////////
 
         "import/no-extraneous-dependencies": ["error", {"devDependencies": true, "optionalDependencies": false}],
         "import/prefer-default-export": ["off"],
-    }
+    },
+    "overrides": [
+        {
+            "files": ["**/*.test.js"],
+            "env": {
+                "jest": true
+            },
+            "rules": {
+                "max-len": "off",
+                "no-unused-expressions": "off",
+                "no-useless-computed-key": "off", // flow does not support non-string property keys https://github.com/facebook/flow/issues/380
+                "react/no-find-dom-node": "off"
+            }
+        },
+        {
+            "files": ["**/stories.js"],
```

Comment:
what about `stories.ts` ?

Reply:
541b86d

Reply:
just saw, that all stories in adverity components are eslint ignored anyway. should i keep the change?

Reply:
We do not ignore stories in components.
https://github.com/eslint/eslint/issues/8429